hieradata/nodes/dev-anthrax.yaml

```diff
@@ -0,0 +1 @@
+ocf::packages::postfix::manage: false
```

Comment:
Can we do this based on tagged class instead of hostname so that production and `dev-` servers get the same configuration with less duplication? (unless overridden by hostname of course)